src/List.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import List from './List';
+
+describe('List', () => {
+  const onClickDelete = jest.fn();
+
+  function renderList(tasks) {
+    return render((
+      <List
+        tasks={tasks}
+        onClickDelete={onClickDelete}
+      />
+    ));
+  }
+
+  context('without task', () => {
+    it('rendrs message', () => {
```

Comment:
오타가 있네요 ㅎㅎ
```suggestion
    it('renders message', () => {
```

그리고 조금 더 명확하게 무슨 메시지인지 적어주면 좋을 것 같아요~

Reply:
앗! ㅋㅋㅋ 저도 지금 발견해서 고치고 있었어요. 
좀 더 명확한 메시지와 함께 적겠습니다!